testing/apps/details/Dockerfile.sidecar

```diff
@@ -12,11 +12,11 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-FROM amalgam8/a8-sidecar:alpine
+FROM amalgam8/a8-sidecar:latest
```

Comment:
Because otherwise the sidecar image doesn't pick up my go code changes, it pulls the alpine image.